opensrp-app/src/main/java/org/smartregister/sync/helper/LocationServiceHelper.java

```diff
@@ -37,7 +40,7 @@
 
     public static final String LOCATION_STRUCTURE_URL = "/rest/location/sync";
     public static final String CREATE_STRUCTURE_URL = "/rest/location/add";
-    public static final String DISTRICT_LOCATIONS_URL = "/location/facilities/";
+    public static final String COMMON_LOCATIONS_SERVICE_URL = "/location/facilities/";
```

Comment:
@cozej4 I think we have to change this content to match the URL declared in the Controller

Reply:
I think this was missed.